src/index.js

```diff
@@ -1 +1,46 @@
 import './style.css';
+import heart from '../assets/heart.png';
+
+const fullPath = 'https://api.tvmaze.com/shows';
+
+// Loading images heart like
+const imgHearts = document.getElementsByClassName('cardboard-heart-image');
+
+for (let i = 0; i < imgHearts.length; i += 1) {
+  imgHearts[i].src = heart;
+}
+
+const getAllShows = async () => {
+  try {
+    const response = await fetch(fullPath);
+    if (!response.status) {
+      throw new Error('Failed to create a post.');
+    }
+    const leaderboardResponse = await response.json();
+    return leaderboardResponse;
+  } catch (error) {
+    return error.message;
+  }
+};
+
+const result = await getAllShows();
+const qtyShows = result.length;
+
+function getRandomInt(max) {
+  return Math.floor(Math.random() * max);
+}
```

Comment:
#### This utility function is working great 🍾 
I think it would be a good idea to take advantage of ES6 syntax and arrow functions (e.g. const getRandomInt = (max) => { //some code });